internal/arista/arista.go

```diff
@@ -77,3 +79,28 @@ func (n *Namer) Fabric(index int) (string, error) {
 	}
 	return fmt.Sprintf("Fabric%d", index+1), nil
 }
+
+// Port is an Arista implementation of namer.Port.
+func (n *Namer) Port(pp *namer.PortParams) (string, error) {
+	var nameBuilder strings.Builder
+	nameBuilder.WriteString("Ethernet")
+	if pp.SlotIndex != nil {
+		nameBuilder.WriteString(fmt.Sprintf("%d/", (*pp.SlotIndex)+3))
+	}
+	nameBuilder.WriteString(fmt.Sprintf("%d", pp.PortIndex))
```

Comment:
nit: put the "/" in the next printf.

Reply:
Not sure what you meant by this, because there was no "sprintf" after this one. I did collapse the logic in the next if-statement, though, which may have been what you meant.